src/server/app.js

```diff
@@ -9,17 +9,17 @@ const express       = require('express'),
 const app = express();
 
 // initialize DB
-require('./db/mongoDB');
-
+require('./db/mongodb');
 
+app.set('view engine', 'html');
 // uncomment after placing your favicon in /public
 //app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
 app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.text());
 app.use(bodyParser.json({ type: "application/vnd.api+json" }));
-app.use(express.static(path.join(__dirname, '../public')));
+app.use(express.static(path.join(__dirname, '../../dist')));
```

Comment:
Why is the public folder now set to to dist?
Dist is short for distribution and typically means a folder containing trans-piled code from the src build. The public folder on the other hand would be outside either src or dist and contain public resources we dont mind exposing to the browser. Also, we don't need to transpile this project into a distribution version.